xcube/core/store/fs/impl/fs.py

```diff
@@ -96,7 +96,7 @@ def get_fs_params_schema(cls) -> JsonObjectSchema:
                     default=True,
                 ),
                 requester_pays=JsonBooleanSchema(
-                    description='If "RequesterPays" buckets are supported.',
+                    description='If "RequesterPays" buckets are supporteendd.',
```

Comment:
What? `supporteendd` ?

Reply:
Not only accidentally commited, accidentally changed. Seems I wrote 'end' when the wrong window had the focus. I change it back.